core/config/v2/docs/secrets.toml

```diff
@@ -0,0 +1,48 @@
+[Database]
+# URL is the PostgreSQL URI to connect to your database. Chainlink nodes require Postgres versions >= 11. See
+# [Running a Chainlink Node](https://docs.chain.link/docs/running-a-chainlink-node/#set-the-remote-database_url-config) for an example.
+#
+# Environment variable alternative:
+# `CL_DATABASE_URL=postgresql://user:pass@localhost:5432/dbname?sslmode=disable`
+URL = "postgresql://user:pass@localhost:5432/dbname?sslmode=disable" # Example
+# BackupURL is where the automatic database backup will pull from, rather than the main DATABASE_URL. It is recommended
+# to set this value to a read replica if you have one to avoid excessive load on the main database.
+#
+# Environment variable alternative:
+# `CL_DATABASE_BACKUP_URL=postgresql://user:pass@localhost:5432/backupdbname?sslmode=disable`
+BackupURL = "postgresql://user:pass@localhost:5432/backupdbname?sslmode=disable" # Example
+# AllowSimplePasswords skips the password complexity check.
+#
+# TODO need an env var?
+AllowSimplePasswords = false # Default
```

Comment:
If someone is using env vars for their secrets, is it a burden to have to create a `secrets.toml` just to use this field?

Reply:
Probably everything in this file should have an ENV var alternative.